golang/cosmos/x/swingset/keeper/keeper.go

```diff
@@ -76,6 +78,61 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
 
+func (k Keeper) GetBeansPerUnit(ctx sdk.Context) map[string]sdk.Uint {
```

Comment:
Doc comment for public methods, here and below. Can simply refer to the intended interface method.